scripts/shared/clusters.sh

```diff
@@ -85,12 +89,13 @@ function create_kind_cluster() {
     kind create cluster $image_flag --name=${cluster} --config=${RESOURCES_DIR}/${cluster}-config.yaml
     kind_fixup_config
 
-    ( deploy_cni; ) &
+    ( deploy_cluster_capabilities; ) &
     if ! wait $! ; then
-        echo "Failed to deploy custom CNI, removing the cluster"
+        echo "Failed to deploy cluster capabilities, removing the cluster"
         kind delete cluster --name=${cluster}
         return 1
     fi
+
```

Comment:
```suggestion
```
Think you left this here ;)

Reply:
Lol, yes, that was my litter. Cleaned it up in the latest. ;)